roles/openshift_manage_node/defaults/main.yml

```diff
@@ -1,9 +1,9 @@
 ---
 # openshift_manage_node_is_master is set at the play level.
 openshift_manage_node_is_master: False
-
-# Default is to be schedulable except for master nodes.
-l_openshift_manage_schedulable: "{{ openshift_schedulable | default(not openshift_manage_node_is_master) }}"
-
+# openshift_has_dedicated_nodes is set on play level, don't taint node if its the only node
+openshift_has_dedicated_nodes: False
```

Comment:
Why are we adding something like this?

Reply:
This would avoid setting taints when all nodes are masters (e.g. CI runs 'all-in-one' installation)